src/openforms/formio/components/translations.py

```diff
@@ -7,12 +7,23 @@ def translate_options(
     enabled: bool,
 ) -> None:
     for option in options:
+        if "openForms" not in option:
+            continue
+
         if not (translations := option.get("openForms", {}).get("translations")):
             continue
```

Comment:
```suggestion
        if not (translations := option["openForms"].get("translations")):
            continue
```
can be simplified/made more readable